addons/core/translations/resources/en-us.yaml

```diff
@@ -554,6 +554,9 @@ target:
     cannot-connect:
```

Comment:
I feel like the name of this translation should change and be more specific since we have one that's now `could-not-connect`